TESTS.md

```diff
@@ -0,0 +1,284 @@
+# UI tests
```

Comment:
I tried to include a lot of info here, including code snippets for Jest, RTL
Let me know if it's overkill 😅 
I can remove some of the spying, mocking examples

Reply:
I thin it's fine, but let's put a describe.skip into them so they're not run by the code itself.

Reply:
omg, I forgot about that 🤦 
I can piggy-back this in my other test ticket